pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/patch/DeletePatchFactory.java

```diff
@@ -0,0 +1,70 @@
+package org.broadinstitute.dsm.model.patch;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import lombok.NonNull;
+import org.apache.commons.lang3.NotImplementedException;
+import org.broadinstitute.dsm.db.dao.ddp.tissue.TissueDao;
+import org.broadinstitute.dsm.db.dao.ddp.tissue.TissueSMIDDao;
+import org.broadinstitute.dsm.model.NameValue;
+import org.broadinstitute.dsm.statics.DBConstants;
+import org.broadinstitute.dsm.util.NotificationUtil;
+
+public class DeletePatchFactory {
```

Comment:
This DeletePatchFactory decides what kind of patch class should be assigned to the request. The PatchRoute checks the patch and if it is a Delete Patch instance, this class will create the appropriate type of `Delete` patcher